File: DAC_API/Models/DTO/User/UserResponseDTO.cs
using System;

namespace DAC_API.Models.DTO.User {
    public class UserResponseDTO {
        public int IdUser { get; set; }
        public string Username { get; set; }
        public string Email { get; set; }
        public string? ProfilePicture { get; set; }
        public DateTime? CreatedAt { get; set; }
        public int? Points { get; set; }
    }
}

File: DailyAdventureChallenge/app/src/main/java/com/example/dailyadventurechallenge/data/dto/User/UserResponseDTO.kt
package com.example.dailyadventurechallenge.data.dto.User

data class UserResponseDTO(
    val idUser: Int,
    val username: String,
    val email: String,
    val profilePicture: String? = null,
    val createdAt: String? = null,
    val points: Int? = null
)